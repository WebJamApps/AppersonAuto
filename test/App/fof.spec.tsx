import React from 'react';
import renderer from 'react-test-renderer';
import FourOhFour from '../../src/App/404';
import { BrowserRouter } from 'react-router-dom';

describe('/404', () => {
  it('renders the component', () => {
    const result = renderer.create(<BrowserRouter><FourOhFour /></BrowserRouter>).toJSON();
    expect(result).toMatchSnapshot();
  });
});
