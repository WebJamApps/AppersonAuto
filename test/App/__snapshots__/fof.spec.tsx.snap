// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/404 renders the component 1`] = `
<div
  className="page-content"
>
  <div
    className="fof"
  >
    404 - Page not available
  </div>
</div>
`;
